addons/website_mass_mailing/static/src/js/website_mass_mailing.js

```diff
@@ -86,6 +89,15 @@ publicWidget.registry.subscribe = publicWidget.Widget.extend({
         return this.$target.closest('[data-snippet=s_newsletter_block').data('list-id') || this.$target.data('list-id');
     },
 
+    _displayListDoesNotExist() {
+        const insertedDiv = document.createElement('div');
+        this.$target[0].appendChild(insertedDiv);
+        insertedDiv.outerHTML = `\
+            <div class="alert alert-danger text-center" groups="event.group_event_user">
+                <p class="m-0">This list was removed from the database. Edit the target list or delete this block.</p>
```

Comment:
this string need to be translated
```
_t('This list was removed from the database. Edit the target list or delete this block.');
```

Also, I would not create the element like that, but with `createElement` and `appendChild` (like above) and set the text directly on the element (otherwise you will technically have an XSS for someone who can create translation)

Reply:
Ah, good call. Something didn't feel right about using outerHTML...